### PR TITLE
Added Teeworlds Mod Server Template

### PR DESCRIPTION
**Pull request by getbrainerror**
Added Teeworlds Mod Server Template to install any teeworlds server.

### CODE REVIEW

File: teeworlds-server-mod/teeworlds-server-mod.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    "install": {	
+      "commands": [
+		{
+		  "type": "download",
+		  "files": [
+			"${url}"
+		  ]
+		},
+		{
+          "type": "writefile",
```
Comment (LordRalex) [inline teeworlds-server-mod/teeworlds-server-mod.json]: Is there really a reason to write a file with the args? Why does this need a start.sh file?

Comment (getbrainerror) [reply teeworlds-server-mod/teeworlds-server-mod.json]: Yes it is, as the executable can be any value and vars dont work in here...

Comment (LordRalex) [reply teeworlds-server-mod/teeworlds-server-mod.json]: Then, it seems like the binary name needs to use that variable too? You have it currently hardcoded to the DD name in the start.sh you write.

Comment (getbrainerror) [reply teeworlds-server-mod/teeworlds-server-mod.json]: Thx, committed a fix for that.

**Comment by LordRalex** [comment]
Since exec is a program, I actually wonder if using that as the program, then having the first arg being the executable would work.

**Comment by LordRalex** [comment]
v1 will no longer accept new templates. If you wish to re-open this with v2, we'll be glad to look at it again.

**Pull request closed without merging**